packages/components/RadioGroup/src/RadioButton.tsx

```diff
@@ -81,6 +81,8 @@ export const RadioButton = compose<IRadioButtonType>({
         accessibilityLabel: ariaLabel ? ariaLabel : content,
         accessibilityState: { disabled: state.disabled, selected: state.selected },
         accessibilityActions: [{ name: 'Select', label: radioButtonSelectActionLabel }],
+        accessibilityPositionInSet: info.buttonKeys.findIndex((x) => x == buttonKey) + 1,
```

Comment:
Playing devils advocate - what happens if findIndex can't find the buttonKey for some reason? What does this return?

Reply:
If findIndex can't find an element that satisfies the testing function, it returns -1. If we add one to that we get zero which is the default value for the native PositionInSet property. So if for some reason it can't find the button key for a radio button, it will assign it zero and the screen reader won't announce anything about the position or size of set for that radio button. But I don't foresee that happening since we pull the button keys from each child directly during each render.